src/libstore/worker-protocol.hh

```diff
@@ -50,6 +50,7 @@ typedef enum {
     wopAddToStoreNar = 39,
     wopQueryMissing = 40,
     wopQueryDerivationOutputMap = 41,
+    wopAddToStoreCANar = 42, // since Nix 3.0
```

Comment:
Shouldn't this be called `wopAddToStoreFromDump`?

Reply:
I figured it's a sibling of `wopAddToStoreNar`, so I've added `CA` to distinguish it, being only capable of producing that kind of store path. `Dump` sounds a bit vague to me, whereas `Nar` is specific.

Reply:
Yeah `wopAddToStoreFromDump` sounds like it isn't always expected nar-formatted objects, which would be confusing.

Reply:
I actually I wonder if we shouldn't just reuse `wopAddToStore` (by checking the daemon protocol) since this fully replaces it.